src/material-experimental/mdc-input/input.spec.ts

```diff
@@ -1163,7 +1171,13 @@ function createComponent<T>(component: Type<T>,
       ...providers
     ],
   }).compileComponents();
+}
 
+function createComponent<T>(component: Type<T>,
+                            providers: Provider[] = [],
+                            imports: any[] = [],
+                            declarations: any[] = []): ComponentFixture<T> {
+  configureTestingModule(component, {providers, imports, declarations});
```

Comment:
Note that I intentionally kept the parameters for `createComponent` untouched. It's out of scope for this PR to refactor how tests create the component.